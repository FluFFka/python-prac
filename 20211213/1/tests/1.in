[(i * 7) % 29 for i in range(16)]
