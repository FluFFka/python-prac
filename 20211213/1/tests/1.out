[0, 4, 5, 6, 7, 11, 12, 13, 14, 18, 19, 20, 21, 26, 27, 28]
